tests/unit/Util/GetoptTest.php

```diff
@@ -14,9 +14,14 @@
 
 /**
  * @small
+ * @covers \PHPUnit\Util\Getopt
  */
 final class GetoptTest extends TestCase
 {
+    /**
+     * @covers \PHPUnit\Util\Getopt::getopt
```

Comment:
Not sure, perhaps the class-level annotation will suffice?

Reply:
Yes, please only use class-level annotations.